src/transformers/models/auto/modeling_auto.py

```diff
@@ -945,6 +947,7 @@
         ("funnel", "FunnelForSequenceClassification"),
         ("gemma", "GemmaForSequenceClassification"),
         ("gemma2", "Gemma2ForSequenceClassification"),
+        ("modernbert", "ModernBertForSequenceClassification"),
```

Comment:
This does not exist yet - I'd love to see this as it allows for training cross-encoder/reranker models.

Reply:
The plan is to support all the standard encoder task heads.